tests/cypress/tests-e2e/drawing.cy.js

```diff
@@ -67,31 +67,48 @@ describe('Drawing module tests', () => {
                 .should((request) => checkKMLRequest(request, [new RegExp(`${regexExpression}`)]))
         }
 
-        // Check that the linestring has the expected number of points
-        function checkDrawnLineString(numberOfPoints) {
-            cy.readWindowValue('drawingLayer')
-                .then((drawingLayer) => drawingLayer.getSource().getFeatures())
-                .should((features) => {
-                    expect(features).to.be.an('Array').lengthOf(1)
-                    const [feature] = features
-                    expect(feature.getGeometry().getType()).to.eq('LineString')
-                    const lineStringCoordinates = feature.getGeometry().getCoordinates()
-                    expect(lineStringCoordinates).to.be.an('Array').lengthOf(numberOfPoints)
-                })
+        function addDecription(description) {
+            cy.get('[data-cy="drawing-style-feature-description"]').type(description)
+            cy.get('[data-cy="drawing-style-feature-description"]').should(
+                'have.value',
+                description
+            )
+            cy.wait('@update-kml').then(() => {
+                cy.readStoreValue('getters.selectedFeatures[0].description').should(
+                    'eq',
+                    description
+                )
+            })
         }
 
-        function checkDrawnPolygon(numberOfPoints) {
+        // we use the description to identify the feature and check its
+        // geometry type, number of points and type (measure or linepolygon)
+        function checkDrawnFeature(description, numberOfPoints, featureType, type) {
             cy.readWindowValue('drawingLayer')
                 .then((drawingLayer) => drawingLayer.getSource().getFeatures())
                 .should((features) => {
-                    expect(features).to.be.an('Array').lengthOf(1)
-                    const [feature] = features
-                    expect(feature.getGeometry().getType()).to.eq('Polygon')
-                    const polygonCoordinates = feature.getGeometry().getCoordinates()
-                    expect(polygonCoordinates).to.be.an('Array').lengthOf(1)
-                    expect(polygonCoordinates[0]).to.be.an('Array').lengthOf(numberOfPoints)
+                    console.log(description)
+                    console.log(features)
```

Comment:
This looks like debug logs.

If the logs are importants to use during tests, use cy.log and a small description of the content of the log.
If those are debug logs, you should remove them.